addons/hr_expense/models/res_config.py

```diff
@@ -8,6 +8,22 @@ class HrExpenseConfigSettings(models.TransientModel):
 
     alias_prefix = fields.Char('Default Alias Name for Expenses')
     alias_domain = fields.Char('Alias Domain', default=lambda self: self.env["ir.config_parameter"].sudo().get_param("mail.catchall.domain"))
+    group_analytic_accounting = fields.Boolean(string='Analytic Accounting',
+        implied_group='analytic.group_analytic_accounting')
+    group_uom = fields.Boolean("Units of Measure",
+        implied_group='product.group_uom')
+    expense_alias_email = fields.Boolean(string='Let your employees record expenses by email')
+    module_project = fields.Boolean(string="Project")
+    module_sale = fields.Boolean(string="Customer Billing")
+
+    @api.model
+    def get_expense_alias_email(self, fields):
```

Comment:
No need to define fields with **get_default** and **set_default** methods. Name the field like **default_foo** and set the **bar.settings.config** as **default_model** on the field definition.